llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -100,6 +100,16 @@ static cl::opt<bool> ForceMinByValParamAlign(
              " params of device functions."),
     cl::init(false));
 
+static auto GetUpsizedNumEltsAndEltVT = [](unsigned OldNumElts, EVT OldEltVT) {
```

Comment:
Given that it's not inside of the function, this does not have to be a lambda. A function would do.